osu.Game/Screens/Play/Player.cs

```diff
@@ -61,6 +61,7 @@ public class Player : ScreenWithBeatmapBackground, IProvideCursor
 
         public CursorContainer Cursor => RulesetContainer.Cursor;
         public bool ProvidingUserCursor => RulesetContainer?.Cursor != null && !RulesetContainer.HasReplayLoaded.Value;
+        protected float BackgroundOpacity => 1 - (float)DimLevel;
```

Comment:
never used?

Reply:
It is being used by the storyboard still, because storyboardContainer is still a part of Player